src/amo/components/CollectionManager/index.js

```diff
@@ -143,7 +148,8 @@ export class CollectionManagerBase extends React.Component<
         createCollection({
           ...payload,
           defaultLocale: siteLang,
-          includeAddonId: router.location.query.include_addon_id,
+          // $FLOW_FIXME: query parameter values are string, not number.
```

Comment:
Same thing here: this code should have always treated the query string param as a string. Sigh, javascript. Could you file a bug to fix this?